### PR TITLE
61x upgrade ownership bugfix

### PR DESCRIPTION
**Pull request by domenicbove**
# Description

When setting mode, we must also set owner and group or else the ownership is set to root

Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Running thru upgrade with customer

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] Any variable changes have been validated to be backwards compatible

### CODE REVIEW

File: tasks/upgrade_component.yml [hunk 1]
```diff
@@ -15,6 +15,8 @@
     remote_src: true
     dest: "/tmp/upgrade/{{ service_name }}/{{ item | basename }}-{{timestamp}}"
     mode: 0640
+    owner: "{{ owner }}"
+    group: "{{ group }}"
```
Comment (erikgb) [inline tasks/upgrade_component.yml:19]: Nitpick, but it seems like it is common practice to list the owner and group properties before the mode.

Comment (domenicbove) [reply tasks/upgrade_component.yml:19]: ok good suggestion!

**Review by erikgb — APPROVED** [review]
You can consider to change the order of properties (nitpick), but LGTM.

File: tasks/upgrade_component.yml [hunk 2]
```diff
@@ -14,9 +14,9 @@
     src: "{{ item }}"
     remote_src: true
     dest: "/tmp/upgrade/{{ service_name }}/{{ item | basename }}-{{timestamp}}"
+    owner: "{{ config_owner }}"
```
Comment (erikgb) [inline tasks/upgrade_component.yml:17]: Why this change of variable name?

**Review by erikgb — COMMENTED** [review]
Just a question, but generally LGTM.

File: tasks/upgrade_component.yml (hunk 2 above)
Comment (domenicbove) [reply tasks/upgrade_component.yml:17]: I didn't like the variable being the same as the argument
